### PR TITLE
PWG/EMCAL: Adding MC Rejection for EPOS/other background

### PR DESCRIPTION
**Pull request by aschmier**
PWG/EMCAL: Adding macro for MC Rejection for EPOS/other background

### CODE REVIEW

**Comment by FriederikeBock** [comment]
+test

File: PWG/EMCAL/EMCALtasks/AliEmcalRejectMCBackground.h
Comment (mfasDa) [inline PWG/EMCAL/EMCALtasks/AliEmcalRejectMCBackground.h:175]: Attention: having [] in comments is actually a ROOT extension for dynamic arrays reserving space (in this case of size fnHeaders) in the streamer. Since you are using a std::vector you don't make use of that, better remove, though without pointer it will probably be ignored by rootcling.

File: PWG/EMCAL/macros/AddTaskRejectMCBackground.C
Comment (mfasDa) [inline PWG/EMCAL/macros/AddTaskRejectMCBackground.C:1]: This macro should in principle just call the static add function you implemented in the analysis task. Here you basically duplicate the code.

**Review by mfasDa — CHANGES_REQUESTED** [review]
Particularly the constants and the rootcling directives must be removed before merging. The rest is cosmetic stuff.

**Review by mfasDa — CHANGES_REQUESTED** [review]
Code wise it is fine now but you should have squashed the two commits into one. Normally github has a functionality that codeowners (corresponding to approvers in this case) can do it via the webinterface - which is how it is usually done in O2, but the AliPhysics project in github is configured in such an idiotic way that - unlike for O2 for example - all the useful extensions are inaccessible to the reviewers which can only approve or reject commits. So sadly I cannot squash-merge via the webinterface and you have to do it via the command line.

**Comment by mfasDa** [comment]
+1

**Pull request merged**
